### PR TITLE
add different operation

### PR DESCRIPTION
**Pull request by Nziranziza**
- add multiplication,subtraction
- add division and addition
- add functionality for user to select an operation
- add functionality to set default operation

[finishes #168136125]
Screen Shot 2019-08-30 at 10 33 56

### CODE REVIEW

File: src/answerField.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default class Answer {
         this.isSubmitted = true;
       }
     });
-    this.answerField.focus();
+    this.focus();
   }
```
Comment (elvisiraguha) [inline src/answerField.js]: harya ubu ukurikije uko byari bimeze hano hari igihindutse cyane??

File: src/helper/formatOperation.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// @flow
+export default (operation): string => {
+  if (operation === '/') {
+    return ':';
```
Comment (elvisiraguha) [inline src/helper/formatOperation.js]: is " : " the good we can do? I was thinking this " &divide; " html sign

Comment (Nziranziza) [reply src/helper/formatOperation.js]: That is ok! we can do that

File: src/answerField.js (hunk 1 above)
Comment (Nziranziza) [reply src/answerField.js]: This provides us with a way of accessing this method outside of the class. Otherwise you can't do that

Comment (elvisiraguha) [reply src/answerField.js]: okay, thats great

File: src/helper/formatOperation.js (hunk 2 above)
Comment (elvisiraguha) [reply src/helper/formatOperation.js]: yeah

Comment (elvisiraguha) [reply src/helper/formatOperation.js]: so are you creating the other PR?

**Pull request merged**